src/main/webapp/META-INF/hudson.remoting.ClassFilter

```diff
@@ -0,0 +1 @@
+org.jvnet.localizer.Localizable
```

Comment:
What was the purpose of this addition? `Localizable` [was already whitelisted in core](https://github.com/jenkinsci/jenkins/blob/d50004fa4e1f798b9c879881b0a44d76d2cfd0dd/core/src/main/resources/jenkins/security/whitelisted-classes.txt#L142).

Reply:
Oh, that's our negligence. Thanks for reminding.